File: attDisciplinas/src/main/java/com/example/attDisciplinas/model/classesmodel/Classes.java
package com.example.attDisciplinas.model.classesmodel;

import jakarta.persistence.*;

@Entity
@Table
public class Classes {

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Integer id;
    @Column(unique = true, nullable = false, length = 30)
    private String className;
    @Column(unique = true, nullable = false, length = 30)
    private String professorName;

    public Integer getId() {
        return id;
    }

    public void setId(Integer id) {
        this.id = id;
    }

    public String getClassName() {
        return className;
    }

    public void setClassName(String className) {
        this.className = className;
    }

    public String getProfessorName() {
        return professorName;
    }

    public void setProfessorName(String professorName) {
        this.professorName = professorName;
    }
}

File: SergioAtt/app/src/main/java/com/api/splashscreen/model/Classes.kt
package com.api.splashscreen.model

class Classes {

    private var id: Int? = null
    private var className: String = ""
    private var professorName: String = ""

    fun getId(): Int? {
        return this.id
    }

    fun setId(id: Int?) {
        this.id = id
    }

    fun getClassName(): String {
        return this.className
    }

    fun setClassName(className: String) {
        this.className = className
    }

    fun getProfessorName(): String {
        return this.professorName
    }

    fun setProfessorName(professorName: String) {
        this.professorName = professorName
    }
}